what4/src/What4/Utils/StringLiteral.hs

```diff
@@ -171,10 +171,24 @@ stringLitIndexOf (Char16Literal x) (Char16Literal y) k =
     Just n  -> toInteger n + toInteger k
 
 stringLitIndexOf (Char8Literal x) (Char8Literal y) k =
-  case BS.findSubstring y (BS.drop (fromIntegral k) x) of
+  case bsFindSubstring y (BS.drop (fromIntegral k) x) of
     Nothing -> -1
     Just n  -> toInteger n + toInteger k
 
+-- | Get the first index of a substring in another string,
+--   or 'Nothing' if the string is not found.
+--
+--   Copy/pasted from the old `bytestring` implementation because it was
```

Comment:
They have a suggestion in the deprecation warning to use `breakSubstring` instead, is that not appropriate here? https://github.com/haskell/bytestring/blob/a80c4261fafe5365d91e14e09e0f05f4b28684cc/Data/ByteString.hs#L1414

Reply:
Well, `breakSubstring` doesn't compute the desired index, so it has to be postprocessed, and it's not completely trivial, as seen below.  I think the deprecation is silly, CF https://github.com/haskell/bytestring/issues/180